pkg/apis/workload/v1alpha1/types.go

```diff
@@ -85,6 +85,18 @@ const (
 	// the label is removed. They then set the placement state to "Unbound".
 	ClusterResourceStateLabelPrefix = "state.workload.kcp.dev/"
 
+	// InternalSyncerViewAnnotationPrefix is the prefix of the annotation
+	//
+	//   diff.syncer.internal.kcp.dev/<sync-target-name>
+	//
+	// on upstream resources storing the diff between the upstream resource
+	// and the view of the resource as it should be seen by the Syncer in the given SyncTarget.
+	//
+	// The format is a Json Merge Patch.
+	//
+	// TODO(davidfestal): use sync-target-uid instead of sync-target-name
```

Comment:
why the intermediate step?

Reply:
the annotation is not experimental. We cannot easily change its format?

Reply:
Oh, doc is wrong here. I'll update that.

Reply:
I rebased and updated the doc

Reply:
Fixed